src/shared-components/segmentedControl/style.ts

```diff
@@ -0,0 +1,65 @@
+import styled from '@emotion/styled';
+import { buttonReset } from 'src/utils/styles/buttonReset';
+
+import { COLORS, SPACER } from '../../constants';
+import { style as TYPOGRAPHY_STYLE } from '../typography';
+
+type SegmentItemProps = {
+  active: boolean;
+  key: number;
+  onClick: () => void;
+  width: number;
+};
+
+type IndicatorProps = {
+  width: number;
+  transform: string;
+};
+
+export const SegmentsContainer = styled.div`
+  width: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: flex-start;
+  position: relative;
+  border-radius: 80px;
+  background-color: ${COLORS.purple10};
+  padding: ${SPACER.xsmall};
+  border: 5px solid ${COLORS.purple10};
+`;
+
+export const SegmentItem = styled.button<SegmentItemProps>`
+  ${buttonReset};
+  ${TYPOGRAPHY_STYLE.body};
+  color: ${COLORS.purple70};
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  cursor: pointer;
+  margin: 0;
+  border-radius: 80px;
+  background-color: ${COLORS.purple10};
+  width: ${({ width }) => `${width}%`};
+  &:focus {
+    outline: none;
```

Comment:
Ben will probably be able to provide more guidance re: styling for different states, but there's a new `focus` variable you can use here to start playing around with how the component would work with only keyboard controls:

```
&:focus {
  outline: none;
  box-shadow: ${BOX_SHADOWS.focus};
}

Reply:
ah yeah, thanks for calling that out @michaeljaltamirano but we should apply the new focus styling to this: unsure on how it will react between the two options `focus` and `focusInner` but would it be possible to get a screenshot of both with the two scenarios below. These are how it looks with designs but unsure how it will with live code

![image](https://user-images.githubusercontent.com/28582716/94479045-2e6a9a00-01a2-11eb-80b2-5c9a277df4cb.png)

Reply:
@heyamykate I ended up getting out of being underwater this cycle--can you re-tag me for review once the keyboard controls are in place?

Reply:
@heyamykate It looks like there's some CSS finessing to match the above:

Active item has focus state:
<img width="556" alt="Screen Shot 2020-10-07 at 10 20 55 AM" src="https://user-images.githubusercontent.com/13544620/95351608-03a9d100-0887-11eb-856d-d1db9533a4be.png">

Inactive item has focus state:
<img width="543" alt="Screen Shot 2020-10-07 at 10 20 48 AM" src="https://user-images.githubusercontent.com/13544620/95351622-06a4c180-0887-11eb-9b55-d43b6ddc8ac7.png">

Reply:
@heyamykate The focused state looks perfect for the active item.  However, if an inactive item adjacent to the active item, it still has a small piece cutoff:
<img width="525" alt="Screen Shot 2020-10-20 at 8 58 35 AM" src="https://user-images.githubusercontent.com/53094193/96613213-88e3ab80-12b3-11eb-9aeb-4227ad373696.png">

If you add a z-index, it should look like below:
<img width="524" alt="Screen Shot 2020-10-20 at 9 01 36 AM" src="https://user-images.githubusercontent.com/53094193/96613282-9dc03f00-12b3-11eb-802c-eccf3ebc7f0e.png">